<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TxDetails</class>
 <widget class="QDialog" name="TxDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1433</width>
    <height>1044</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Transaction Details</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLabel" name="labelHash">
     <property name="text">
      <string>0000000000000000000000000000000000000000000000</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="labelNote">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>This appears to be a normal payment transaction.</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Raw Transaction Hex</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Total value out:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string># Outputs</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="labelLockTime">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Transaction To String:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="labelNumIn">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="labelValueOut">
     <property name="text">
      <string>0.0000,0000</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Lock Time:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="labelNumOut">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0" colspan="2">
    <widget class="QTextBrowser" name="textBrowserHex">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string># Inputs</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="2">
    <widget class="QTextBrowser" name="textBrowserTx">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="lineWrapMode">
      <enum>QTextEdit::NoWrap</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Hash:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QTreeWidget" name="treeWidgetDecoded">
     <property name="font">
      <font>
       <family>Noto Mono</family>
      </font>
     </property>
     <attribute name="headerMinimumSectionSize">
      <number>50</number>
     </attribute>
     <attribute name="headerDefaultSectionSize">
      <number>300</number>
     </attribute>
     <column>
      <property name="text">
       <string>Type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Details</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>Decoded from transaction outputs:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
